onnxruntime/core/session/lora_adapters.cc

```diff
@@ -62,15 +62,16 @@ namespace {
 struct DataTransfer {
   std::unique_ptr<IExecutionProvider> ep;
   std::unique_ptr<IDataTransfer> data_transfer;
+  bool is_dml = false;
   Status CopyTensor(const Tensor& src, Tensor& dst) const {
     return data_transfer->CopyTensor(src, dst);
   }
   Status Sync() const {
-#if USE_DML
```

Comment:
What is the meaning of `one package`?
If this means that we are going to have to same macro for CUDA and DML presence, then I do not understand how it would work. CUDA libraries depend on the presence of NVIDIA card and drivers, they would not load without it.
Thereas DML functions on a wide variety of hardware, the first and foremost case is onboard GPUs from intel and alike.